src/components/Account/ScreenConnected.js

```diff
@@ -80,13 +80,12 @@ function AccountScreenConnected({ wallet }) {
           >
             <IdentityBadge
               entity={wallet.account}
-              label={shortenAddress(wallet.account)}
               compact
               badgeOnly
               css={`
                 cursor: pointer;
               `}
-              networkType={getNetworkType()}
+              networkType={getNetworkName()}
```

Comment:
We should pass the `chainId` in here to get the name

Reply:
This is outdated, now the code looks like this: `networkType={networkEnvironment.legacyNetworkType}` :)